policy/modules/contrib/cloudform.te

```diff
@@ -164,10 +164,13 @@ optional_policy(`
 ')
 
 optional_policy(`
+    sysnet_domtrans_dhcpc(cloud_init_t)
     sysnet_domtrans_ifconfig(cloud_init_t)
     sysnet_read_dhcpc_state(cloud_init_t)
     sysnet_dns_name_resolve(cloud_init_t)
     sysnet_filetrans_cloud_net_conf(cloud_init_t)
+    sysnet_filetrans_dhcpc_pid(cloud_init_t, "dhclient.pid")
+    sysnet_filetrans_dhcpc_pid(cloud_init_t, "dhclient.lease")
```

Comment:
With sysnet_domtrans_dhcpc(cloud_init_t) in place, is a file transition still needed?

Reply:
no, thank you